docs/finder-content-item.md

```diff
@@ -0,0 +1,61 @@
+# The Finder Content Item Format
+
+A Finder Content Item is a specialisation of the [Content Item](https://github.com/alphagov/content-store/blob/master/doc/content_item_fields.md). This guide explains what goes in the details hash of the ContentItem and why.
+
+The Finder Content Item is used by Finder Frontend to render the Finder page. Most of what it uses goes in the `details` hash.
+
+# The `details` hash
+
+## `beta`
+
+A boolean. Required.
+
+A flag used to decide if the Beta banner should be rendered.
+
+## `beta_message`
+
+A string. Required.
+
+Can contain HTML. If `beta` is true, `beta_message` will be passed to the beta banner.
+
+## `document_noun`
+
+A string. Required.
+
+The lowercase singular version of whatever format the Finder is using. For example: [`/cma-cases`](https://www.gov.uk/cma-cases) has a `document_noun` of `case`, [`/aaib-report`](https://www.gov.uk/aaib-reports) has a `document_noun` of `report`.
+
+## `document_type`
+
+A string. Required.
+
+[snake_case](http://en.wikipedia.org/wiki/Snake_case) string which tells Finder Frontend what doctype to limit the search to in Rummager. It must match the name of the file describing the doctype [in Rummager](https://github.com/alphagov/rummager/tree/master/config/schema/default/doctypes).
```

Comment:
Given that it needs to match the doctype in Rummager, we probably don't need to stipulate here that should be snake_case.